tests/test_meshing_queries_fdl.py

```diff
@@ -4,7 +4,14 @@
 
 import_filename = examples.download_file("mixing_elbow.msh.h5", "pyfluent/mixing_elbow")
 
+PYTEST_RELATIVE_TOLERANCE = 0.2
 
+
+def pytest_approx(expected):
+    return pytest.approx(expected=expected, rel=PYTEST_RELATIVE_TOLERANCE)
+
+
+@pytest.mark.codegen_required
 @pytest.mark.nightly
 @pytest.mark.fluent_version(">=24.2")
 def test_meshing_utilities(new_mesh_session):
```

Comment:
Not completely fixed, the test is too sensitive to data. Maybe we can test only few functions or move the test in pyfluent testing repo in Fluent.

Reply:
I think we can comment out failing functions because the outputs we are expecting are changing periodically.

Reply:
@hpohekar Feel free to look into that part.